<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraTestClass</class>
 <widget class="QWidget" name="CameraTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraTest</string>
  </property>
  <widget class="QLabel" name="label_camera">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_photo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CameraTest.qrc"/>
 </resources>
 <connections/>
</ui>
